docs/index.md

```diff
@@ -1138,17 +1138,20 @@ Cause Mocha to only run tests matching the given `regexp`, which is internally c
 
 Suppose, for example, you have "api" related tests, as well as "app" related tests, as shown in the following snippet; One could use `--grep api` or `--grep app` to run one or the other. The same goes for any other part of a suite or test-case title, `--grep users` would be valid as well, or even `--grep GET`.
 
+And another option with double quotes: `--grep "groupA|groupB"`.<br>
```

Comment:
`<br>` is not needed here.

Reply:
I tried, there is no `end-of-line` without `<br>` in our markdown file.